specification/v3.3/OSDM-online-api-v3.3.0-draft.yml

```diff
@@ -11121,6 +11246,89 @@ components:
           type: string
           format: uri
           nullable: true
+        pointers:
+          type: array
+          items:
+            $ref: '#/components/schemas/ProblemPointer'
+            nullable: true
+          minItems: 0
+
+    ProblemPointer:
+      type: object
+      additionalProperties: false
+      description: |
+        Problem Details for HTTP APIs (RFC 7807) defines a 'problem detail' as a way
+        to carry machine-readable details of errors in a HTTP response to avoid
+        the need to define new error response formats for HTTP APIs.
+        See: https://tools.ietf.org/html/rfc7807
+      properties:
+        code:
+          description: |
+            optional in version 3.1, will be made mandatory in upcoming releases
+          type: string
```

Comment:
Does it make sense to build extensible enum containing at least the problem codes defined on osdm.io?